needy/serialization.py

```diff
@@ -0,0 +1,39 @@
+import json
+
+from collections import OrderedDict
+
+
+def load_yaml(content):
+    try:
+        import yaml
+    except ImportError:
+        raise NotImplementedError('pyyaml must be installed to load YAML content')
```

Comment:
I don't think this is the right error type. I think the import error is more accurate. It can be made more user-friendly at a higher level, as is already done.